packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx

```diff
@@ -36,19 +33,29 @@ const SuccessAction: React.FC<{
         </div>
       ) : (
         <div>
-          <Button size={'large'} mode={'secondary'} style={{ marginTop: 20 }} onClick={goToDaoPage}>
-            Don't register token
-          </Button>
+          <StyledText name={'body2'} style={{ color: theme.disabled }}>
+            Voice enables gasless (non-binding) governance proposals and votes.
+          </StyledText>
```

Comment:
the text with the color theme.disabled is almost impossible to read, is this intentional?

Reply:
nice catch, it should be theme.disabledContent